### ISSUE TITLE
Updating from 7.3.10 to 7.3.11 throws ReferenceError: returntrue is not defined

### ISSUE THREAD

**Issue by philiiiiiipp** [styleguidist/react-styleguidist#1181]
**Current behavior**

After updating from 7.3.10 to 7.3.11 I see this error when going to the our styleguidist website.

```
ReferenceError: returntrue is not defined
    in xt (created by ve)
    in footer (created by ve)
    in main (created by ve)
    in div (created by ve)
    in ve (created by Styled(ve))
    in Styled(ve) (created by t)
    in t
This may be due to an error in a component you are overriding, or a bug in React Styleguidist.

If you believe this is a bug, please submit an issue.
```

the console is also full of them:

```
main.bundle.js:12 Uncaught ReferenceError: returntrue is not defined
    at ye (main.bundle.js:12)
    at Ie (main.bundle.js:12)
    at xt (main.bundle.js:12)
    at _d (main.bundle.js:61)
    at Cd (main.bundle.js:61)
    at jh (main.bundle.js:61)
    at Bh (main.bundle.js:61)
    at HTMLUnknownElement.e (main.bundle.js:48)
    at Object.l (main.bundle.js:48)
    at v (main.bundle.js:48)
ye @ main.bundle.js:12
Ie @ main.bundle.js:12
xt @ main.bundle.js:12
_d @ main.bundle.js:61
Cd @ main.bundle.js:61
jh @ main.bundle.js:61
Bh @ main.bundle.js:61
e @ main.bundle.js:48
l @ main.bundle.js:48
v @ main.bundle.js:48
xh @ main.bundle.js:61
Nh @ main.bundle.js:61
km @ main.bundle.js:61
Em @ main.bundle.js:61
xm @ main.bundle.js:61
bm @ main.bundle.js:61
Vh @ main.bundle.js:61
(anonymous) @ main.bundle.js:61
jm @ main.bundle.js:61
Bm @ main.bundle.js:61
Vm.render @ main.bundle.js:61
(anonymous) @ main.bundle.js:61
Om @ main.bundle.js:61
Ym @ main.bundle.js:61
render @ main.bundle.js:61
u @ main.bundle.js:32
(anonymous) @ main.bundle.js:32
T @ main.bundle.js:1
n @ main.bundle.js:1
(anonymous) @ main.bundle.js:32
T @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
main.bundle.js:61 The above error occurred in the <xt> component:
    in xt (created by ht)
    in article (created by ft)
    in ft (created by Styled(ft))
    in Styled(ft) (created by ht)
    in ht (created by It)
    in section (created by Bt)
    in Bt (created by Styled(Bt))
    in Styled(Bt) (created by It)
    in It (created by Ut)
    in section (created by Lt)
    in Lt (created by Styled(Lt))
    in Styled(Lt) (created by Ut)
    in Ut (created by It)
    in section (created by Bt)
    in Bt (created by Styled(Bt))
    in Styled(Bt) (created by It)
    in It (created by Ut)
    in section (created by Lt)
    in Lt (created by Styled(Lt))
    in Styled(Lt) (created by Ut)
    in Ut (created by t)
    in main (created by ve)
    in div (created by ve)
    in ve (created by Styled(ve))
    in Styled(ve) (created by t)
    in t

React will try to recreate this component tree from scratch using the error boundary you provided, t.
(anonymous) @ main.bundle.js:61
jd @ main.bundle.js:61
Xd.a.function.a.componentDidCatch.r.callback @ main.bundle.js:61
Up @ main.bundle.js:61
Vp @ main.bundle.js:61
Wp @ main.bundle.js:61
Ld @ main.bundle.js:61
Th @ main.bundle.js:61
e @ main.bundle.js:48
l @ main.bundle.js:48
v @ main.bundle.js:48
Ph @ main.bundle.js:61
Am @ main.bundle.js:61
km @ main.bundle.js:61
Em @ main.bundle.js:61
xm @ main.bundle.js:61
bm @ main.bundle.js:61
Vh @ main.bundle.js:61
(anonymous) @ main.bundle.js:61
jm @ main.bundle.js:61
Bm @ main.bundle.js:61
Vm.render @ main.bundle.js:61
(anonymous) @ main.bundle.js:61
Om @ main.bundle.js:61
Ym @ main.bundle.js:61
render @ main.bundle.js:61
u @ main.bundle.js:32
(anonymous) @ main.bundle.js:32
T @ main.bundle.js:1
n @ main.bundle.js:1
(anonymous) @ main.bundle.js:32
T @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
main.bundle.js:61 The above error occurred in the <xt> component:
    in xt (created by ve)
    in footer (created by ve)
    in main (created by ve)
    in div (created by ve)
    in ve (created by Styled(ve))
    in Styled(ve) (created by t)
    in t

React will try to recreate this component tree from scratch using the error boundary you provided, t.
```
I double verified, it works with 7.3.10 but not with 7.3.11.

**To reproduce**

I am not sure what triggered this other than the update. The examples seem to work fine.

We are using:
```
styled-components@4.0.2
react@16.5.2
```

Let me know what other information I can provide.

**Expected behavior**

Should show the website.

**Comment by stonebk** [comment]
This only appears to be an issue with `webpack@4.22.0`. Using `webpack@4.21.0` seems to work.

**Comment by ammmze** [comment]
> This only appears to be an issue with `webpack@4.22.0`. Using `webpack@4.21.0` seems to work.

Pinning webpack back to `4.21.0 also worked for me. (Also had to pin buble to `0.19.4`...currently `0.19.5` is causing a different [issue](https://github.com/Rich-Harris/buble/issues/159))

**Comment by eemeli** [comment]
Pinning webpack isn't working for me. For a proper fix (which works at least with react-styleguidist 8.0.0), you need to pin its `markdown-to-jxs` dependency to 6.7.4 or earlier, as its 6.8 release added an (undocumented!) output minification step that's causing this issue.

The upstream issue probablyup/markdown-to-jsx#224 was closed, probably incorrectly. Not really sure what's the original source of the problem though.

**Comment by probablyup** [comment]
Hi folks, please try markdown-to-jsx 6.8.2

**Comment by yorthehunter** [comment]
Pinning `markdown-to-jsx` to `6.8.2` worked for me. 

FYI, here's what it looks like in my `package.json` (I'm using Yarn):

```
  "resolutions": {
    "markdown-to-jsx": "6.8.2"
  },
```

**Comment by probablyup** [comment]
@sapegin this can be closed now - definitely fixed with 6.8.2
